activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -462,7 +462,8 @@ def execute_grouped_calculation(operation, column_name, distinct) # :nodoc:
         group_columns = group_aliases.zip(group_fields)
 
         column = aggregate_column(column_name)
-        column_alias = column_alias_tracker.alias_for("#{operation} #{column_name.to_s.downcase}")
+        column_name = column.try(:name) || connection.visitor.compile(column)
```

Comment:
We need to get a `name` if `column` is an `Arel::Attribute` (instead of just `connection.visitor.compile(column)`), because otherwise the `compile` method will also include a table name and the test at 
https://github.com/rails/rails/blob/f6e36c90aa484b00538154c41886a33a1e2c9c8c/activerecord/test/cases/calculations_test.rb#L240-L244 

will fail with undefined column `sum_accounts_credit_limit`. Which is unexpected column name, I think.